test/fixtures/qunit/invalid.js

```diff
@@ -5,21 +5,24 @@ QUnit.module( 'Example' );
 QUnit.test( '.foo()', function ( assert ) {
 	const x = 'bar';
 
-	// eslint-disable-next-line qunit/no-assert-equal
+	// eslint-disable-next-line qunit/no-loose-assertions
 	assert.equal( x, 'bar' );
 
+	// eslint-disable-next-line qunit/no-loose-assertions
+	assert.notEqual( x, 'bar' );
+
 	assert.expect( 3 );
 
 	if ( !x ) {
 		// eslint-disable-next-line qunit/no-early-return
 		return;
 	}
 
-	// eslint-disable-next-line qunit/no-assert-ok
-	assert.ok( x );
```

Comment:
Did you mean to removee this one? Plain use of `ok()` seems like it'd still be useful to warn against (more common than the negated one we still test for). I think `no-loose-assertions` warns for both.

Reply:
Yes - I figured we only needed one, and leaving the negated one sort of asserts that we don't get the no-negated-ok error as well.